webapp/titanembeds/templates/site_layout.html.j2

```diff
@@ -16,6 +16,13 @@
     <meta name="apple-mobile-web-app-title" content="TitanEmbeds">
     <meta name="application-name" content="TitanEmbeds">
     
+    <!--Specify icons for Apple devices (can also be used as fallback for Android)-->
+    <link rel="apple-touch-icon" href="https://titanembeds.com/static/img/titanembeds_square.png">
```

Comment:
For the URLs, could you make it a relative urls?
Remember that other people can run the Titan project and they have different domains. Just drop the https://titanembeds.com from the URl and it would be great!